pants.toml

```diff
@@ -173,7 +173,7 @@ args = ["--external-sources"]
 args = ["-i 2", "-ci", "-sr"]
 
 [pytest]
-args = ["--no-header", "-vv"]
```

Comment:
Why this change? We want `-vv` because it's annoying when PYtest doesn't show the whole diff so you can't figure out what's wrong. That has happened several times before we set this.

Reply:
well, I did add the `-vv` flag while I was testing this feature and then reverted it back once I had my tests working. I can re-add it again, I do indeed find it annoying that Pytest doesn't give the whole diff when tests fail.

note that this setting only applies when running locally, the ci-specific `.toml` has the `-vv` flag set.